index.html

```diff
@@ -679,6 +679,15 @@ <h3>Options</h3>
 properties.
       </p>
     </section>
+    <section>
+      <h3>HTTP Headers</h3>
+      <p>
+All requests and responses sent to/from the VC API MUST contain the following header:
```

Comment:
As written, this would preclude proper use of HTTP methods that do not have bodies, so I think we need this tweak:

```suggestion
All requests and responses sent to/from the VC API that include entity bodies MUST
contain the following header:
```